app/tests/data_factory.py

```diff
@@ -1074,10 +1074,9 @@ def create_sample(
             The data sample
         '''
         if db_create and create_asset_file:
-            raise ValueError(
-                'Cannot have db_create=True and create_asset_file=True, db_create=True implicitely '
-                'create the asset file when the "file" attribute is present'
-            )
+            # the asset file are automatically created by the model when creating the model
+            # therefore here correct the `create_asset_file` to false to avoid issues
+            create_asset_file = False
```

Comment:
That does the trick, I guess.
But probably it would be more intuitive, instead of forcibly set `create_asset_file = False` to do something like in the following (even if the code would grow a bit), just a suggestion, what do you think?

```python
        if db_create and create_asset_file:
            return super().create_sample(
                sample,
                name=name,
                item=item,
                db_create=db_create,
                required_only=required_only,
                **kwargs
            )
        
        else:
            return super().create_sample(
                sample,
                name=name,
                item=item,
                db_create=db_create,
                required_only=required_only,
                create_asset_file=create_asset_file,
                **kwargs
            )
```

Reply:
of course with the comment, that you added above, also added somewhere in this snippet

Reply:
But I can also live with your one-liner ;-)